scout/server/blueprints/cases/templates/cases/cases.html

```diff
@@ -126,10 +126,11 @@ <h5 class="modal-title" id="sangerModalLabel">Sanger validations to evaluate:</h
     <thead>
       <tr>
         <th class="col-xs-2">{{ group_name|capitalize }} cases</th>
-        <th class="col-xs-2">Status</th>
-        <th class="col-xs-2">Analysis date</th>
+        <th class="col-xs-1">Status</th>
+        <th class="col-xs-1">Analysis date</th>
         <th class="col-xs-2">Link</th>
         <th class="col-xs-2">Sequencing</th>
+        <th class="col-xs-2">Track</th>
```

Comment:
Thinking as a user, maybe it would be more clear if you rename "Track" to something else, pipeline maybe?

Reply:
We agreed on track in the meeting last week... If we are to change it we need to talk to the others. Could we keep `Track` while discussing this?

Reply:
Let's keep track!